### PR TITLE
Tweak display of illustrations in branding radio labels

### PR DESCRIPTION
**Pull request by quis**
Before | After
---|---
image | image

### CODE REVIEW

**Comment by quis** [comment]
I’ve updated this again to keep the two column grid (so the label text doesn’t wrap) but without the illustrations getting overly-large:

image

File: app/assets/stylesheets/components/radios-with-images.scss [hunk 1]
```diff
@@ -7,19 +7,25 @@
   @include govuk-responsive-margin(4, "right");
 
   padding-left: 0;
+  cursor: pointer;
```
Comment (samuelhwilliams) [inline app/assets/stylesheets/components/radios-with-images.scss:10]: I don't think we want this here, do we?

This style gets applied when the JS is loaded, and that's what actually makes it clickable. So with this if the CSS loaded and the JS failed, we'd show a pointer but clicking the image wouldn't do anything.

Comment (samuelhwilliams) [inline app/assets/stylesheets/components/radios-with-images.scss:10]: Do we definitely want this?

The image is only clickable via JS support. If the CSS loads and the JS fails then people might click on the image but it won't select the input.

Comment (quis) [reply app/assets/stylesheets/components/radios-with-images.scss]: Good point, removed in 700539e759db44c2f22105a5e4b2f90bd8627eec

File: app/assets/javascripts/radios-with-images.js [hunk 2]
```diff
@@ -14,7 +14,6 @@
     this.start = function ($radioImage) {
 
       var radioImageNode = $radioImage.get(0);
-      console.log(radioImageNode);
```
Comment (samuelhwilliams) [inline app/assets/javascripts/radios-with-images.js:17]: 👀 💥

**Review by samuelhwilliams — APPROVED** [review]
👍

**Pull request merged**